index.html

```diff
@@ -170,9 +170,19 @@ <h2>
         </h2>
         <p>
           A <dfn>push subscription</dfn> is a message delivery context established between the
-          <a>user agent</a> and the <a>push service</a> on behalf of a web application. Each
-          <a>push subscription</a> is associated with a <a>service worker registration</a> and a
-          <a>service worker registration</a> has at most one <a>push subscription</a>.
+          <a>user agent</a> and the <a>push service</a> on behalf of a web application.
+        </p>
+        <p>
+          A [=push subscription=] has an associated <dfn for="push subscription">scope</dfn>, which
+          is a [=/URL=].
+        </p>
+        <p>
+          A [=push subscription=] is considered to have a <dfn for=
+          "push subscription">window-accessible scope</dfn> when its [=push subscription/scope=] is
+          a [=/list=] of [=list/size=] 1 and [=push subscription/scope=][0] is the empty string.
```

Comment:
Here the scope seems to be a list of string, but right above it's defined as a URL struct, which looks like a conflict.

Reply:
Indeed, this should look at the scope's path instead of scope.